etna/packages/etna-js/upload/api/upload_api.jsx

```diff
@@ -8,4 +8,4 @@ export const postAuthorizeUpload = (base_url, project_name, bucket_name, file_pa
 
 export const postUploadStart = (upload_url, request) => json_post(upload_url, { action: 'start', ...request });
 export const postUploadCancel = (upload_url, request) => json_post(upload_url, { action: 'cancel', ...request });
-export const postUploadBlob = (upload_url, request) => form_post(upload_url, { action: 'blob', ...request });
+export const postUploadBlob = (upload_url, request, checkStatus= true) => form_post(upload_url, { action: 'blob', ...request }, checkStatus);
```

Comment:
Useful when custom logic is needed around certain status codes; allows optionally that the promise succeed with the entire response object.